extensions/skyportal/static/js/components/Filter.jsx

```diff
@@ -0,0 +1,401 @@
+          <Card className={classes.root}>
+            <CardContent>
+              {group && stream && (
+                <Typography
+                  className={classes.title}
+                  color="textSecondary"
+                  gutterBottom
+                >
+                  Group: <Link to={`/group/${group.id}`}>{group.name}</Link>
+                  <br />
+                  {/* Group id: {group.id} */}
+                  Stream: {stream.name}
+                </Typography>
+              )}
+            </CardContent>
+            {filter_v && filter_v.catalog && (
+              <CardActions>
+                {/* <Button size="small">Delete</Button> */}
+                <FormControlLabel
+                  style={{ marginLeft: 5 }}
+                  control={
+                    <Switch
+                      checked={filter_v.active}
+                      size="small"
+                      onChange={handleActive}
+                      name="filterActive"
+                    />
+                  }
+                  label="Active"
+                />
+                <FormControl className={classes.formControl}>
+                  <InputLabel id="alert-stream-select-required-label">
+                    Active version
+                  </InputLabel>
+                  <Select
+                    labelId="alert-stream-select-required-label"
+                    id="alert-stream-select"
+                    value={filter_v.active_fid}
+                    onChange={handleFidChange}
+                    className={classes.selectEmpty}
+                  >
+                    {filter_v.fv.map((fv) => (
+                      <MenuItem key={fv.fid} value={fv.fid}>
+                        {fv.fid}: {fv.created_at.slice(0, 19)}
+                      </MenuItem>
+                    ))}
+                  </Select>
+                  {/* <FormHelperText>Required</FormHelperText> */}
+                </FormControl>
+              </CardActions>
+            )}
+          </Card>
+        </Grid>
+        {/* /!* Filter stats go here? *!/ */}
+        {/* <Grid item sm={12} md={9}> */}
+        {/* </Grid> */}
+      </Grid>
+
+      <br />
+
+      <Accordion
+        expanded={panelExpanded === "panel"}
+        onChange={handlePanelChange("panel")}
+      >
+        <AccordionSummary
+          expandIcon={<ExpandMoreIcon />}
+          aria-controls="panel-content"
+          id="panel-header"
+        >
+          <Typography className={classes.heading}>Save new version</Typography>
+        </AccordionSummary>
+        <AccordionDetails className={classes.accordion_details}>
+          <form onSubmit={handleSubmit(onSubmitSaveFilterVersion)}>
+            <Grid container spacing={2}>
+              <Grid item sm={12} md={10}>
+                <TextareaAutosize
+                  rowsMax={10000}
+                  rowsMin={6}
+                  placeholder="Filter definition (please refer to the docs)"
```

Comment:
```suggestion
                  placeholder="Filter definition (please refer to the docs at https://fritz-marshal.org/doc/user_guide.html#alert-filters-in-fritz)"
```
Maybe put a link to the relevant docs in this placeholder?

Reply:
Done